src/styles/_compare.scss

```diff
@@ -48,3 +48,12 @@ table, th, tr, td {
     padding: 0px 8px;
   }
 }
+
+.arrow {
```

Comment:
Glad you added the arrow. I think we should use the same one on the Character and Stage filters for consistency.